test/rdf-components-mocha.js

```diff
@@ -104,4 +122,30 @@ describe('rdf components', function() {
             });
         }).should.eventually.eql(cynthia);
     });
+    it("should round trip a graph through ntriples", function() {
```

Comment:
Round trips the data, like the previous test, but this goes through an rdf-ntriples component an its way.